mloop/utilities.py

```diff
@@ -464,6 +465,25 @@ def stop(self):
         '''
         pass
 
+class ParameterScaler(skp.MinMaxScaler):
+    def __init__(self, min_boundary, max_boundary, *args, **kwargs):
+        '''
+        Class to scale the parameters of the gaussian process. 
+        All parameters are mapped (by default) between [0, 1] using min and max boundaries
+        '''
+        if len(min_boundary) != len(max_boundary):
+            raise("The minimum and maximum boundary arrays must have the same lengths")
```

Comment:
This should be a `ValueError`. It's also often very helpful for debugging to print out information about the inputs.

```suggestion
            raise ValueError(
                "The minimum and maximum boundary arrays must have the same lengths but "
                f"min_boundary had length {len(min_boundary)} while max_boundary had length "
                f" {len(max_boundary)}."
            )
```